frontend-react/src/pages/getting-started/testing-facilities/AccountRegistrationGuide.tsx

```diff
@@ -0,0 +1,335 @@
+                            >
+                                SimpleReport
+                            </a>
+                            , a PRIME project that ReportStream closely
+                            collaborates with. If you have questions about any
+                            step of the registration process, contact us at
+                            reportstream@cdc.gov.
+                        </p>
+                    </div>
+                </div>
+            </section>
+
+            <section>
+                <h3 className="font-body-lg border-top-1px border-ink margin-top-8 margin-bottom-6 padding-top-1">
+                    Register for a ReportStream account
+                </h3>
+                <ol className="usa-process-list rs-process-list__documentation">
+                    <li className="usa-process-list__item">
+                        <h4 className="usa-process-list__heading">
+                            Contact ReportStream
+                        </h4>
+                        <p className="margin-top-05">
+                            Send an email to reportstream@cdc.gov. Use the
+                            example text below as a template for your message:
+                        </p>
+                        <blockquote className="rs-blockquote__documentation">
+                            <p>
+                                To: reportstream@cdc.gov
+                                <br />
+                                Subject: Register a new account: organization or
+                                testing facility
+                            </p>
+
+                            <p>Hi,</p>
+
+                            <p>
+                                I'd like to get a new user account set up for
+                                ReportStream.
+                            </p>
+
+                            <p>
+                                Name:
+                                <br />
+                                Job title:
+                                <br />
+                                Organization:
+                                <br />
+                                Email address:
+                                <br />
+                                Phone number:
+                                <br />
+                            </p>
+                        </blockquote>
+                    </li>
+                    <li className="usa-process-list__item">
+                        <h4 className="usa-process-list__heading">
+                            Accept the terms of service
+                        </h4>
+                        <ul className="margin-top-2">
+                            <li>
+                                A ReportStream representative will process your
+                                registration request and respond via email with
+                                a link to an online form.{" "}
```

Comment:
Did you mean to have `{" "}` for every item in this list? Seems better to have some style markup than depend on React to inject it.

Reply:
@TomNUSDS looks like the linter is adding that, I didn't any `{" "}` to the markup. Is that something we should change with the linter?

Reply:
Oh, interesting. I think the linter might be trying to preserve multiple spaces in the final output (which seems silly for html)? @sean-usds any thoughts?

Reply:
Actually I _kind of_ get why it's doing it, the linter is adding line breaks around things like `<a>` while also removing empty spaces around the text. It needs those `{" "}` to make the flow around other text work.

Without it you'd have:

```
Lorem ipsum
  <a href="/foo">
    LINK-TEXT
  </a>
dolor sit amet
```

That would translate to:

> Lorem ipsumLINK-TEXTdolor sit amet